Gems/AtomLyIntegration/AtomViewportDisplayIcons/Code/Source/AtomViewportDisplayIconsSystemComponent.cpp

```diff
@@ -131,7 +131,7 @@ namespace AZ::Render
             return;
         }
 
-        // Find our icon, falling back on a grey placeholder if its image is unavailable
+        // Find our icon, falling back on a gray placeholder if its image is unavailable
```

Comment:
Both spellings are correct so you are just entering holy war territory here :)

Reply:
I checked and 'gray' is used more in American English which is what I've been told to use in other reviews 😝 (behavior not behaviour) so I think this is the right spelling for O3DE

Reply:
🤣 I'm not aware of a locale specification for our codebase so I think I went with default-American when I wrote this, I'm not fussed either way tbh.

Reply:
The other reason I fixed this is the Visual Assist spell checker I have flagged it up so I thought I might as well fix it 😝 I've had people moan at me for my British English before so I'm now hypersensitive about it 😛